core/chains/evm/txmgr/eth_broadcaster.go

```diff
@@ -152,13 +152,6 @@ func (eb *EthBroadcaster) Start(ctx context.Context) error {
 			return errors.Wrap(err, "EthBroadcaster could not start")
 		}
 
-		if eb.config.EvmNonceAutoSync() {
```

Comment:
This had to move into txmgr to avoid autosyncing nonce again e.g. on key change, reset